pybamm/models/submodels/interface/lithium_plating/base_plating.py

```diff
@@ -132,27 +147,48 @@ def _get_standard_concentration_variables(self, c_plated_Li, c_dead_Li):
 
         return variables
 
-    def _get_standard_reaction_variables(self, j_stripping):
+    def _get_standard_reaction_variables(self, j_plating, j_stripping, j_dead_lithium):
         """
         A private function to obtain the standard variables which
         can be derived from the lithum stripping interfacial reaction current
+
         Parameters
         ----------
+        j_plating : :class:`pybamm.Symbol`
+            The lithium plating interfacial reaction current.
         j_stripping : :class:`pybamm.Symbol`
-            The net lithium stripping interfacial reaction current.
+            The lithium stripping interfacial reaction current (from reactions only).
+        j_dead_lithium : :class:`pybamm.Symbol`
+            The lithium stripping interfacial reaction current (from dead lithium).
+
         Returns
         -------
         variables : dict
             The variables which can be derived from the plated lithium thickness.
         """
-        # Set scales to one for the "no plating" model so that they are not required
-        # by parameter values in general
+        j_plating_av = pybamm.x_average(j_plating)
         j_stripping_av = pybamm.x_average(j_stripping)
+        j_dead_lithium_av = pybamm.x_average(j_dead_lithium)
+
+        j_plating_net = j_plating + j_stripping + j_dead_lithium
```

Comment:
@DrSOKane if I understand correctly this is incorrect and the plating current (the one that goes into the total current conservation equation) should just be plating + stripping?